Informative error msg for unsupported shells on show() (#1481)
spacetelescope/jdaviz: CHANGES.rst

Other Changes and Additions
---------------------------

> - Add friendly message when trying to load Jdaviz outside of Jupyter [#1481]

Reviewer: "friendly" is too subjective.

```suggestion
- Added a more informative error message when trying to load Jdaviz outside of Jupyter [#1481]
```

- Fixed a bug where having Subset breaks coordinates information display
  in image viewers. [#1472]

> Other Changes and Additions

Reviewer: Why is this removed? Is it on purpose?

Author: Nope, not sure how that got in there; I'll revert that
